pkg/services/accesscontrol/roles.go

```diff
@@ -195,28 +169,58 @@ const (
 
 	ldapAdminEdit = "fixed:ldap:admin:edit"
 	ldapAdminRead = "fixed:ldap:admin:read"
+)
+
+var (
+	once sync.Once
 
-	provisioningAdmin = "fixed:provisioning:admin"
+	// FixedRoles provides a map of permission sets/roles which can be
+	// assigned to a set of users. When adding a new resource protected by
+	// Grafana access control the default permissions should be added to a
+	// new fixed role in this set so that users can access the new
+	// resource. FixedRoleGrants lists which built-in roles are
+	// assigned which fixed roles in this list.
+	FixedRoles sync.Map
```

Comment:
once we move roles definitions to the packages, we can make this private and remove the InitFixedRole function